.semaphore/semaphore-scheduled-builds.yml

```diff
@@ -152,8 +152,10 @@ blocks:
       matrix:
       - env_var: ARCH
         values: [ "arm64", "ppc64le" ]
+      - env_var: LDFLAGS
```

Comment:
I think this is actually the wrong spot to put this - these are our scheduled builds which are dev builds and we _do_ want the debug symbols present for these builds. 

We only want to strip the binaries for the released images, which are built through a different workflow at the moment.

Here's where our release logic exists: https://github.com/projectcalico/calico/blob/master/hack/release/pkg/builder/builder.go#L374-L388

It calls through to the `release-build` target in each directory, passing in a limited set of env vars.

Reply:
Reverted this changes.